configs/foreman/3.1.yaml

```diff
@@ -0,0 +1,492 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :3.1.0:
+    :redmine_version_id: 1408
```

Comment:
```suggestion
    :redmine_version_id: 1488
```

Reply:
Thanks, where I can find the ID for Redmine? I tried to find it yesterday.

Reply:
Done, it's now
>  :redmine_version_id: 1488

Reply:
I took it from the URL on https://projects.theforeman.org/projects/foreman/roadmap. Really it should use the API which can tell you the ID but that's some coding that needs to be done in tool belt itself.